bin/java

```diff
@@ -17,8 +17,11 @@ source "${JVM_COMMON_BUILDPACK_DIR}/lib/inventory.sh"
 # shellcheck source=lib/java_properties.sh
 source "${JVM_COMMON_BUILDPACK_DIR}/lib/java_properties.sh"
 
-install_java_with_overlay() {
+install_openjdk() {
```

Comment:
This PR is marked as skip-changelog, though seeing as it adds a new API and deprecates an old one, I think it should still have a changelog entry? (Given there are library consumers as it were)

Reply:
That's a good point! I'll add it. I might put the README changes in here as well instead of a second PR too.